addons/account/tests/test_company_branch.py

```diff
@@ -253,3 +253,13 @@ def test_branch_should_keep_parent_company_currency(self):
         })
         self.env['account.chart.template'].try_loading('generic_coa', company=root_company.child_ids[0], install_demo=False)
         self.assertEqual(root_company.currency_id, root_company.child_ids[0].currency_id)
+
+    def test_branch_user_tax_methods(self):
```

Comment:
This is more of a unit test than a feature test (we are just testing the private method). I think, we should instead test the user flow.

Also, we are not asserting anything.

Reply:
Thanks :)

The implied assertion was that there was no AccessError raised.

I have modified the test so it uses the highest flow without adding a tour test (a call to route `/website_sale/combo_configurator/get_data` ) and added one assertion (that without the fix would not be reached because of the AccessError).